src/main/resources/db/migration/V4__alter_project_table.sql

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Shouldn't this script be named 'V1__XXXXXX' since there's no other script files in the repo? 🤔

Reply:
ok, but if there are other V1, does that mess it up?

Reply:
If thats the case you will have to change this file name when you pull in the latest changes from the main branch, so just be careful! Just keep an eye on the versioning in the migration script folder and pick the next available one